changelog.md

```diff
@@ -7,6 +7,7 @@
   methods in `cai_causal_graph.causal_graph.CausalGraph` and its subclasses, as none of them use it.
 - Improved the `cai_causal_graph.time_series_causal_graph.TimeSeriesCausalGraph.get_minimal_graph` method in
   `cai_causal_graph.time_series_causal_graph.TimeSeriesCausalGraph` to avoid a recursion issue.
+- Added `cai_causal_graph.causal_graph.CausalGraph.add_edges_from_paths` convenience method to add edges from paths.
```

Comment:
Suggest "Added `cai_causal_graph.causal_graph.CausalGraph.add_edges_from_paths` convenience method to `cai_causal_graph.causal_graph.CausalGraph` in order to add ..." Just so the class will be there too. Otherwise, it just shows up as `add_edges_from_paths` when it renders.